### DISCUSSION TITLE
Resource section of the site

### DISCUSSION THREAD

**Discussion by BekahHW** [Virtual-Coffee/virtualcoffee.io#816] [General Resources]
We'd love to develop our resource section of the site that support some of the common topics we hear about during coffees and in slack. If you have ideas or resources, drop them below.

**Comment by jdwilkin4** [comment]
Here is my list of resources that have been shared a few times in the VC slack 😄 

- [You're Not Just Tired: The Psychology of Burnout](https://www.youtube.com/watch?v=-XYK8ulQId0)
- [CSS for JavaScript Developers](https://css-for-js.dev/)
- [Exercisim](https://exercism.org/) 
- [OWASP foundation](https://owasp.org/www-project-top-ten/)
- https://csswizardry.gumroad.com/l/perfect-devtools
